examples/webgl_loader_gcode.html

```diff
@@ -48,7 +48,7 @@
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				container.appendChild( renderer.domElement );
 
-				var controls = new OrbitControls( camera, renderer.domElement );
+				new OrbitControls( camera, renderer.domElement );
```

Comment:
I'm not sure if we want to set this kind of example for the users.
I know LGTM complains about it, but in reality having access to the controls variable offers many possibilities of customization and having an empty constructor also feels wrong to me.

Reply:
Agreed. Instead, maybe add `controls.min/maxDistance` -- or `controls.min/maxZoom` in the case of orthographic. Or `controls.enableZoom = false`.

That could be done in a separate PR.

Reply:
I agree that it's weird. But I think we have used this pattern before?

Reply:
I think I’m more worried about what new users might infer from that, other then the fact that it is completely safe to do so. 

I like WestLangley idea as well.

Reply:
We have certainly used this pattern:
```js
new FooLoader()
	.load( url, ... )
```

I do not have a problem using the same pattern with `OrbitControls`, but I do have a problem allowing the user to zoom without limitation.

Reply:
> could be done in a separate PR

if you want I can remove controls commit from PR, I mean it is separate since you know... premonition.

Reply:
there - gone.